testing/yarn/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=yarn
+pkgver=0.19.1
+pkgrel=0
+pkgdesc="Fast, reliable, and secure dependency management for Node.js"
+url="https://yarnpkg.com/"
+arch="noarch"
+license="BSD-2"
+depends="nodejs"
+source="https://github.com/yarnpkg/$pkgname/releases/download/v$pkgver/$pkgname-v$pkgver.tar.gz
```

Comment:
Could you please update this to use `https://yarnpkg.com/downloads/$pkgver/yarn-v$pkgver.tar.gz`? In the Yarn project, we're trying to avoid hard-coding GitHub URLs for downloads.